tests/www/employee_record_views/test_summary.py

```diff
@@ -22,6 +22,12 @@ def test_access_granted(self):
         response = self.client.get(self.url)
         assert response.status_code == 200
 
+    def test_check_job_seeker_infos(self):
+        # Must have access
+        self.client.force_login(self.user)
+        response = self.client.get(self.url)
+        self.assertContains(response, self.job_application.job_seeker.jobseeker_profile.birth_place)
```

Comment:
Inclure un peu de markup autour de l’élément dont on souhaite vérifier la présence limite le risque de faux positifs.
```suggestion
        profile = self.job_application.job_seeker.jobseeker_profile
        self.assertContains(
            response,
            f"<li>À : {profile.birth_place} ({profile.birth_place.department_code})</li>",
            count=1,
        )
```

Reply:
Voir un snapshot pour prendre toutes les info ("Etat civil" et "Situation du salarié") d'un coup et pas trop se prendre la tête avec le markup ?

Reply:
Pas le courage de faire un `for_snapshot` pour `JobApplicationWithCompleteJobSeekerProfileFactory` (et son coutournement de profil de demandeur d'emploi) :see_no_evil: